apps/myaccount/src/components/account-recovery/account-recovery.tsx

```diff
@@ -176,41 +187,50 @@ export const AccountRecoveryComponent: FunctionComponent<AccountRecoveryProps> =
-                                                    data-testid={ `${testId}-settings-section-email-recovery` }
-                                                />
+                                                    onAlertFired={onAlertFired}
+                                                    data-testid={`${testId}-settings-section-security-questions-component`}/>
                                             )
-                                            : null
-                                    }
+                                            : null}
+                                    </List.Item><List.Item className="inner-list-item">
+                                    {hasRequiredScopes(
+                                        featureConfig?.security,
+                                        featureConfig?.security?.scopes?.read,
+                                        allowedScopes
+                                    ) &&
+                                    isFeatureEnabled(
+                                        featureConfig?.security,
+                                        AppConstants.FEATURE_DICTIONARY
+                                            .get("SECURITY_ACCOUNT_RECOVERY_EMAIL_RECOVERY")
+                                    ) &&
+                                    (isNotificationRecoveryEnabled || isUsernameRecoveryEnabled)
+                                        ? (
+                                            <EmailRecovery
+                                                onAlertFired={onAlertFired}
+                                                data-testid={`${testId}-settings-section-email-recovery`}/>
+                                        )
+                                        : null}
                                 </List.Item>
+                                </>
                             </List>
-                        </SettingsSection>
+                            : <EmphasizedSegment>
+                                <Placeholder fluid>
+                                    <Placeholder.Header image>
+                                        <Placeholder.Line />
+                                        <Placeholder.Line />
+                                    </Placeholder.Header>
+                                    <Placeholder.Header image>
+                                        <Placeholder.Line />
+                                        <Placeholder.Line />
+                                    </Placeholder.Header>
+                                    <Placeholder.Header image>
+                                        <Placeholder.Line />
+                                        <Placeholder.Line />
+                                    </Placeholder.Header>
+                                </Placeholder>
+                            </EmphasizedSegment>
                     )
-                    : null
+                    }
```

Comment:
Possible formatting error here. Lets reformat the ternary this way https://github.com/wso2/identity-apps/blob/master/docs/DEVELOPER.md#ternary-expressions

Reply:
Addressed with 1997335a9368d5f07d34a552c95a7c0045369740